package.json

```diff
@@ -9,6 +9,7 @@
     "express": "^4.17.1",
     "express-ws": "^4.0.0",
     "http-errors": "^1.7.3",
+    "lodash.random": "^3.2.0",
```

Comment:
if it's not used by the backend, it would be a `devDependency` because it's only needed at build time. Everything in the react app gets bundled

Reply:
resolved and re-opened #36